Source/WebCore/workers/service/server/SWRegistrationDatabase.cpp

```diff
@@ -79,7 +79,16 @@ static String scriptDirectoryPath(const String& directory)
     if (directory.isEmpty())
         return emptyString();
 
-    return FileSystem::pathByAppendingComponents(directory, { "Scripts"_s, scriptVersion });
+    return FileSystem::pathByAppendingComponent(directory, "Scripts"_s);
+}
+
+static String scriptVersionDirectoryPath(const String& directory)
+{
+    auto scriptDirectory = scriptDirectoryPath(directory);
+    if (scriptDirectory.isEmpty())
+        return emptyString();
+
+    return FileSystem::pathByAppendingComponent(scriptDirectory, scriptVersion);
```

Comment:
It is doing a bit more than just removing she and wal files.
We will now delete ./Scripts folder instead of ./Scripts/V1 folder.
This is probably fine but it is worth mentioning this in the commit message.

Reply:
`We will now delete ./Scripts folder instead of ./Scripts/V1 folder.`
This is expected, I should say "delete shm file, wal file, and empty Scripts folder"; will add